regression/cbmc/Struct_Initialization/flexible_array_member.desc

```diff
@@ -0,0 +1,10 @@
+KNOWNBUG
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+^warning: ignoring
+--
+Based on issue #3653.
```

Comment:
Please provide sufficient information in here (and in the commit message!) so that one doesn't have to go through extra hoops to figure out what this is about. Moreover, this isn't the same code as in the bug report and it's not clear whether both assertions in this test now fail or just one of them.